### PR TITLE
add support for multiarch

### PR DESCRIPTION
**Pull request by pkking**
#11 and for https://github.com/lxc/lxc-ci/issues/663

### CODE REVIEW

**Comment by pkking** [comment]
@mudler @monstermunchkin @TommyLike can you help to review

File: main.go [hunk 1]
```diff
@@ -41,6 +41,10 @@ func main() {
 					Name:  "keep",
 					Usage: "Keeps downloaded layers around (useful for debugging)",
 				},
+				cli.StringFlag{
```
Comment (TommyLike) [inline main.go:44]: 1. Specify/specify
2. Can we specify the default value here?

File: api/download.go [hunk 2]
```diff
@@ -23,7 +23,25 @@ type DownloadOpts struct {
 	UnpackMode       string
 }
 
-func DownloadAndUnpackImage(sourceImage, output string, opts *DownloadOpts) error {
+func getTargetImageSha(hub *registry.Registry, repo, tag, arch string) (string, error) {
+	manifestList, err := hub.ManifestList(repo, tag)
```
Comment (TommyLike) [inline api/download.go:39]: Is it possible to have no manifest list for specific image?

**Comment by TommyLike** [comment]
@pkking thanks!

**Pull request closed without merging**